README.md

```diff
@@ -471,7 +471,22 @@ We can combine the two precedent filters:
 The argument `-filter withtest` indicates that only commits with at least one modification on test cases are considered.
 
 
+## Analysis
```

Comment:
Hi @monperrus and @niloofartrg ,

Wouldn't it be better to have this documentation in the [README](https://github.com/kth-tcs/defects4j-repair-reloaded/blob/comrepair-coming/README.md) of [defects4j-repair-reloaded/comrepair-coming](https://github.com/kth-tcs/defects4j-repair-reloaded/blob/comrepair-coming/)?

WDYT?

Reply:
It could be in both. There is never too much documentation in research prototypes :-)